src/main/java/org/fogbowcloud/arrebol/ArrebolController.java

```diff
@@ -136,10 +170,10 @@ public void stop() {
 
     public String addJob(Job job) {
 
-        job.setJobState(JobState.READY);
+        job.setJobState(JobState.QUEUED);
```

Comment:
do we still need the SUBMITTED state?

Reply:
it would be great to document the enum

Reply:
The submitted state is used as the instantiation state of the job.